tpp-app/src/info/Info.js

```diff
@@ -1,26 +1,168 @@
 import React from 'react';
-import { Text, View } from 'react-native';
-import { Card, Button } from 'react-native-elements'
+import { StyleSheet, Text, View, Linking, ScrollView, Image, TouchableOpacity } from 'react-native';
+import padIcon from '../../ios/tppapp/Images.xcassets/icons/pad_icon.png';
+import tamponsIcon from '../../ios/tppapp/Images.xcassets/icons/tampons_icon.png';
+import underwearIcon from '../../ios/tppapp/Images.xcassets/icons/underwear_icon.png';
+import cupIcon from '../../ios/tppapp/Images.xcassets/icons/cup_icon.png';
+import clothPadIcon from '../../ios/tppapp/Images.xcassets/icons/clothpad_icon.png';
+
+const LearnMoreCard = () => {
+    return(
+        <View style={styles.learnMoreCard}>
+            <Text style={styles.productText}>Learn more about The Period Purse</Text>
+            
+            <Text style={styles.learnMoreText}>
+            The Period Purse strives to achieve menstrual equity by providing people 
+            who menstruate with access to free menstrual products, and to reduce the 
+            stigma surrounding periods through public education and advocacy.
+            </Text>
+            <TouchableOpacity
+                style={styles.visitButton}
+                onPress={() => Linking.openURL('https://www.theperiodpurse.com')}
+            >
+                <Text style={{...styles.productText, margin: 10}}>Visit the website</Text>
+            </TouchableOpacity>
+        </View>  
+    )
+}
+
+const PadsCard = () =>{
```

Comment:
imo u can make a general `InfoCard` component that takes it's source & text from `props`, and use it to replace the 5 different `*Card` components. It makes it less readable, but I think you will still be able to tell which card is which based off `props`.

Reply:
Beautiful suggestion! I'll update it.